data/commodities.txt

```diff
@@ -260,7 +358,9 @@ trade
 		"pig iron"
 		"platinum wool"
 		"platinum"
+		"printer heads"
```

Comment:
Not a metal. Maybe an equipment? Also, I think it's `printhead` in the US.

Reply:
I think it's valid for normal printers as well